app/views/pages/index.html.slim

```diff
@@ -7,6 +7,7 @@
         .header-link
           .block.request-access
             = mail_to 'aloha@jellypbc.com', 'Request Early Access', class: "btn btn-secondary"
+          p class="devise__registrationPrompt"
```

Comment:
**request**: place `| Already have an account? #{link_to "Log In", new_user_session_path}` inside a ```div``` instead of the ```p``` to match existing style of code in this file. 

**suggestion**: you can reuse the block div.

example:

```
.block
  | Already have an account? #{link_to "Log In", new_user_session_path}
```

using the block div, the div will need ```padding-bottom: 16px;``` from ```/pages.scss```, so the change in ```/pages.scss``` will need to be reverted

**question**: what does ```class="devise__registrationPrompt``` do? i couldn't figure out why it is needed. am i missing something?

Reply:
great catch! i completely agree with you. i am unsure what that class does, perhaps @dennyluan will be able to answer this.

Reply:
nevermind, i misunderstood what class you are referring to. i believe that class exists in order to import the necessary CSS rules required? i wanted to copy the format of the log in prompt as on the sign up page, and since that included this class, i kept it in to ensure consistency.

Reply:
i meant `login-block` (or any new div class name) instead of `block` in my suggestion above. sorry!